azure-pipelines.yml

```diff
@@ -35,13 +35,38 @@ jobs:
     inputs:
       command: publish
       arguments: '--no-build --nologo --configuration $(BuildConfiguration) --output $(build.ArtifactStagingDirectory)/net6.0 /p:PackageVersion=$(Build.BuildNumber)-development --framework net6.0'
+      publishWebProjects: false
```

Comment:
Why was this disabled?

Reply:
When `true`, this seems to override the `projects` key and specifically looks for a web project itself. This lead to this job building the new project instead of the Demo project, I must assume because it "BlazorWasmDemo" starts with a B and therefore is found first, but this is the only official information I could use: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/dotnet-core-cli-v2?view=azure-pipelines (the comments in the syntax block at the top).